components/filter/filter.js

```diff
@@ -60,6 +63,10 @@ class Filter extends LocalizeCoreElement(RtlMixin(LitElement)) {
 				white-space: nowrap;
 			}
 
+			.d2l-filter-dimension-set-value-text {
+				line-height: unset;
+			}
+
```

Comment:
This is unrelated - I noticed the checkboxes weren't centering properly with the text since applying the `d2l-body-compact` styles last PR.